src/javascript/binary/pages/bet/tick_trade.js

```diff
@@ -153,7 +153,7 @@ var TickDisplay = function() {
                         if (data[i][0] === 'tick') {
                             var tick = {
                                 epoch: parseInt(data[i][1]),
-                                quote: parseFloat(data[i][2])
+                                quote: parseFloat(data[i][2]).toFixed(data[i][2].split('.')[1].length)
```

Comment:
How are we rounding? Note that every underlying has its own specific "number of decimals" setting. Do we have an API call that returns this? If not we need to add one, and then do the rounding properly.